configure.py

```diff
@@ -1492,6 +1492,13 @@ def main():
     else:
       set_trisycl_include_dir(environ_cp)
 
+  set_action_env_var(environ_cp, 'TF_NEED_ROCM', 'ROCm', False)
+  if environ_cp.get('TF_NEED_ROCM') == '1':
```

Comment:
nit: These two `if` statements can be consolidated as one, because neither of them has an `else` branch.

Reply:
I'd like to keep them as is for now. There are additional upcoming ROCm-specific checks / env vars in future PRs. For example, like in CUDA path where it's possible to build TensorFlow with either `nvcc` or `CUDA clang`, we are also working on similar route where it's possible to switch between incumbent `HIP/HCC` toolchain or upcoming `HIP clang` toolchain.

Reply:
@gunan for the time being `HIP/HCC` would be the incumbent toolchain on ROCm and we'll switch to `HIP clang` toolchain once its performance beats the incumbent solution.

I'm working on revising this PR to address comments from all reviewers now and will ping you once it's ready.